modules/programs/qutebrowser.nix

```diff
@@ -50,6 +82,90 @@ in {
       '';
     };
 
+    keyMappings = mkOption {
+      type = types.attrsOf types.str;
+      default = { };
+      description = ''
+        This setting can be used to map keys to other keys. When the key used as dictionary-key is pressed, the binding for the key used as dictionary-value is invoked instead. This is useful for global remappings of keys, for example to map Ctrl-[ to Escape. Note that when a key is bound (via <literal>bindings.default</literal> or <literal>bindings.commands</literal>), the mapping is ignored.
```

Comment:
Please limit the description lines to 80 characters or so for consistency with other modules. The same applies for the other options.